src/run/runner/wall_time/perf/unwind_data.rs

```diff
@@ -0,0 +1,144 @@
+// !!!!!!!!!!!!!!!!!!!!!
+// !!! DO NOT MODIFY !!!
+// !!!!!!!!!!!!!!!!!!!!!
+//
```

Comment:
@not-matthias same here for the decoration, adding a warning below would be good enough